### ISSUE TITLE
Toggle default value is not persisted

### ISSUE THREAD

**Issue by kraftvaerk-chth** [umbraco/Umbraco-CMS#17212]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.2.0

### Bug summary

When fetching content either through webhooks or the content delivery API, it seems that default values are treated as null instead of showing the correct default value configured on the property's data type in backoffice

### Steps to reproduce

Create a content type with a toggle field, ensure the toggle field has a data type with a "initial state" of true
Create a new document of this content type, and publish

When fetching the document through the content delivery API, or when looking at the payload from a webhook, the property value is null

### Expected result / actual result

Actual: The property value is null
Expected: The property value should be "true", since the field is shown as set to true in the backoffice

**Comment by kjac** [comment]
Hi @kraftvaerk-chth,

Sorry for the belated response here.

I can reproduce on V14+ ... this is not directly related to the Delivery API, though, but to the backoffice client. Turns out it does not send the default value to the server at save/publish time:

![image](https://github.com/user-attachments/assets/c569c5ae-40fe-4120-a9ab-75d63d1999fc)

I'll rewrite the issue title a little and see if I can come up with a fix.

**Comment by kjac** [comment]
@LeReinhard Hi 👋 yes, we're currently investigating options for default values as a whole.

**Comment by kjac** [comment]
Hi everyone.

**TL;DR: For the time being, don't rely on default values for toggles.**

We're going to introduce a new concept of "presets" to the backoffice. This will cover the use case of "default toggled" for the toggle, as well as many other, more advanced, scenarios in the future.

We hope to get this done in a non-breaking manner for V15.3, but we may have to postpone it until V16.

**Comment by kjac** [comment]
Future note: The Slider editor should also be using "presets" when they're available. See #17099

**Comment by nielslyngsoe** [comment]
Hi @kraftvaerk-chth

Could you help us with the prioritization of which to implement first by listing the Property Editors where you are missing this?

Thanks in advance

**Comment by nielslyngsoe** [comment]
An update on this matter, the Toggle Preset will be resolved as part of this PR. Which is in the making and is targeting 15.3.

https://github.com/umbraco/Umbraco-CMS/pull/18423

**Comment by madsrasmussen** [comment]
Fixed and part of 15.3 https://github.com/umbraco/Umbraco-CMS/pull/18423
